lib/src/webcrypto/webcrypto.hmac.dart

```diff
@@ -25,7 +25,44 @@ part of 'webcrypto.dart';
 ///  * [JWK] format using [HmacSecretKey.importJsonWebKey].
 ///
 /// A random key can also be generated using [HmacSecretKey.generateKey].
-///
+/// 
+/// **Example**
+/// ```dart
+/// import 'package:webcrypto/webcrypto.dart';
+/// import 'dart:convert';
+/// 
+/// Future<void> main() async {
+///   // Generate a new key using SHA-256 and an optional length parameter.
+///   final key = await HmacSecretKey.generateKey(Hash.sha256, length: 256);
+///   
+///   // Sign the message.
+///   final signature = await key.signBytes(utf8.encode('Hello World!'));
+/// 
+///   // Verify the signature.
+///   final verified = await key.verifyBytes(signature, utf8.encode('Hello World!'));
+///   assert(verified == true, 'Signature should be valid');
+/// 
+///   // Sign a stream of data.
```

Comment:
Let's omit the streaming API...

We don't want to give the impression that its different...

Otherwise, we could do signBytes and verifyStream if we want to be fun.

Or we could explain they produce the same result.

Reply:
Removed signStream and verifyStream

Reply:
It's possible that in all classes that have `xxxStream` and `xxxBytes` we should make a note that they give the same result, given the same list of bytes :D

Though, I'm not sure it's necessary, it's kind of obvious, and we probably point it out a few places too.

But it might be nice to make tutorial style documentation thing about the streaming API, perhaps.